src/main/java/com/ultimatelogger/multiplatform/SwitchableUltimateLogger.kt

```diff
@@ -0,0 +1,6 @@
+package com.ultimatelogger.multiplatform
+
+interface SwitchableUltimateLogger : UltimateLogger {
+
+    var shouldLog: Boolean
```

Comment:
It seems that you've exposed this var in end user classes. I guess we don't want that to be settable by user at any time.